docs/content/concepts/graphql-rpc.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: GraphQL for Sui RPC (Alpha)
+title: GraphQL for Sui RPC (Beta)
```

Comment:
It would be good to add a info or warn admonition at the top of this doc (before the "Headers" section) to ask developers to migrate away from the GraphQL alpha endpoints https://sui-mainnet.mystenlabs.com/graphql and https://sui-testnet.mystenlabs.com/graphql to the new GraphQL beta endpoints of https://graphql.mainnet.sui.io/graphql and https://graphql.testnet.sui.io/graphql due to the schema changes. We will turn off the alpha endpoints soon.